src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/VariantAnnotatorIntegrationTest.java

```diff
@@ -1,342 +1,369 @@
-            args.addReference(new File(reference));
+    // the purpose of VariantAnnotator is to emulate the annotations that HaplotypeCaller and Mutect2 would emit after
+    // local assembly and calculating likelihoods with Pair-HMM.  Thus our basic test is to compare the annotations of
+    // VariantAnnotator to those of Mutect2
+    @Test
+    public void testAgainstMutect2() throws Exception {
+        final File bam = new File(largeFileTestDir + "mutect/dream_synthetic_bams/", "tumor_4.bam");
+        final File mutect2AnnotatedVcf = createTempFile("mutect2-annotated", ".vcf");
+        final File mutect2UnannotatedVcf = createTempFile("mutect2-unannotated", ".vcf");
+        final File reannotatedVcf = createTempFile("reannotated", ".vcf");
+        final File reference = new File(b37Reference);
+
+        // run Mutect2 on an interval that has indels and multiallelics
+        final ArgumentsBuilder mutect2Args = new ArgumentsBuilder()
+                .addInput(bam)
+                .addOutput(mutect2AnnotatedVcf)
+                .addReference(reference)
+                .addInterval(new SimpleInterval("20", 50_000_000, 52_000_000))
+                .addBooleanArgument(StandardArgumentDefinitions.ENABLE_ALL_ANNOTATIONS, true);
+
+        final ArgumentsBuilder mutect2ArgsWithoutAnnotations = new ArgumentsBuilder()
+                .addInput(bam)
+                .addOutput(mutect2UnannotatedVcf)
+                .addReference(reference)
+                .addInterval(new SimpleInterval("20", 50_000_000, 52_000_000))
+                .addBooleanArgument(StandardArgumentDefinitions.DISABLE_TOOL_DEFAULT_ANNOTATIONS, true);
+
+        new Main().instanceMain(makeCommandLineArgs(mutect2Args.getArgsList(), Mutect2.class.getSimpleName()));
+
+        new Main().instanceMain(makeCommandLineArgs(mutect2ArgsWithoutAnnotations.getArgsList(), Mutect2.class.getSimpleName()));
+
+        // annotate the unannotated Mutect2 output
+        final ArgumentsBuilder args = new ArgumentsBuilder()
+                .addInput(bam)
+                .addVCF(mutect2UnannotatedVcf)
+                .addReference(reference)
+                .addOutput(reannotatedVcf)
+                .addBooleanArgument(StandardArgumentDefinitions.ENABLE_ALL_ANNOTATIONS, true);
+
+        runCommandLine(args.getArgsArray());
+
+        // check a representative sampling of annotations
+        final Set<VariantAnnotation> representativeInfoIntegerAnnotations = ImmutableSet.of(new BaseQuality(), new MappingQuality(), new FragmentLength(), new ChromosomeCounts());
+        final Set<VariantAnnotation> representativeInfoStringAnnotations = ImmutableSet.of(new ReferenceBases());
+        final Set<VariantAnnotation> representativeInfoBooleanAnnotations = ImmutableSet.of(new TandemRepeat());
+
+        final Set<GenotypeAnnotation> representativeGenotypeAnnotations = ImmutableSet.of(new DepthPerAlleleBySample(), new DepthPerSampleHC(), new OrientationBiasReadCounts());
```

Comment:
Is AlleleFraction too variable?  That's one that I and other users are very fond of.

Reply:
Nope; added it to the test with no problems.